great_expectations/data_context/data_context/ephemeral_data_context.py

```diff
@@ -41,3 +41,8 @@ def __init__(
 
     def _init_variables(self) -> EphemeralDataContextVariables:
         raise NotImplementedError
+
+    def _save_project_config(self) -> None:
```

Comment:
Is this necessary? Maybe better to throw an error if this is attempted.

Reply:
Good point :) Converting this into an error.

Reply:
@anthonyburdi Keeping this as a logger message would maintain the pattern already set by `CloudDataContext`.  Would it be ok to keep it as a logging message for now?